components/not-found-page/index.js

```diff
@@ -1,13 +1,25 @@
+import PropTypes from 'prop-types';
 import classNames from './not-found-page.module.scss';
 
-export default function index() {
+const Index = ({ errorMsg }) => {
   return (
     <div className={classNames.fullPageContainer}>
```

Comment:
Please add`Head` here as well. 
```
<Head>
  <title>Not Found</title>
</Head>
```

Reply:
@whyDontI has his PR ready for this changes

Reply:
Okay cool.